usr/packages.conf

```diff
@@ -5,10 +5,11 @@ php=https://windows.php.net/downloads/releases/php-7.3.3-Win32-VC15-x64.zip
 apache=https://home.apache.org/~steffenal/VC15/binaries/httpd-2.4.38-win64-VC15.zip
 
 # phpMyAdmin
-*phpmyadmin=https://files.phpmyadmin.net/phpMyAdmin/4.9.5/phpMyAdmin-4.9.5-english.zip
+*phpmyadmin=https://files.phpmyadmin.net/phpMyAdmin/5.2.0/phpMyAdmin-5.2.0-english.zip
 
 # MariaDB 
-# mariadb10.3=https://downloads.mariadb.org/interstitial/mariadb-10.3.13/winx64-packages/mariadb-10.3.13-winx64.zip/from/https%3A//mirrors.nxthost.com/mariadb/?serve
+# mariadb10.3=http://downloads.mariadb.org/rest-api/mariadb/10.0.3/mariadb-10.0.3-winx64.zip
+# mariadb10.5.4=http://downloads.mariadb.org/rest-api/mariadb/10.5.4/mariadb-10.5.4-winx64.zip
```

Comment:
WHy did you commented this line? it hsould be uncommented to be included. Also 10.5.4 is fine as this is a long term version valid until Jun 2025 https://mariadb.com/kb/en/changes-improvements-in-mariadb-105/